control_plane/tools/roles/configure_new_devices/tasks/main.yml

```diff
@@ -93,6 +93,10 @@
       changed_when: true
       no_log: true
 
+    - name: Wait for 10 mins for DHCP to assign IP to devices
+      wait_for:
+        timeout: "{{ dhcp_wait_time }}"
+
```

Comment:
Is the 10 minute wait based on a particular data point? How many nodes/IPs does the 10 minute window allow for?

Reply:
This playbook is scheduled as a cron job that runs daily and this playbook is for additional nodes adding to cluster. All idrac ip assigned by dhcp will be visible in 10minutes in AWX, once TOR switch comes up. In case of any IP is missed for provision, the cron job will pick it up in next run or user can launch idrac_template manually anytime.